modules/core/src/iterators/make-stream/make-node-stream.ts

```diff
@@ -1,4 +1,6 @@
-import {Readable, ReadableOptions} from 'stream';
+import type {ReadableOptions, Readable} from 'stream';
+// Indirection to make things work under "problematic" bundlers like vite and rollup
+import {ReadableStream} from './node-stream-polyfill';
```

Comment:
I think this has broken my bundler, I'm using `parcel@2`. Since package.json does not have `node-stream-polyfill` in dependencies.

Reply:
We are working on providing a fix https://github.com/visgl/loaders.gl/issues/2021 and will release a new release ASAP